pkgs/servers/komga/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchurl
+, lib
+, makeWrapper
+, jdk11_headless
+}:
```

Oh, and this (we prefer `lib` (and `stdenv`, I think) before everything else. I also ordered the rest in order of occurrence in the file, but that really doesn't matter -- feel free to just move `lib`, or reorder the rest as you see fit):

```suggestion
{ lib
, stdenv
, fetchurl
, makeWrapper
, jdk11_headless
}:
```